test/ReactWrapper-spec.jsx

```diff
@@ -385,6 +394,8 @@ describeWithDOM('mount', () => {
     });
 
     it('should not find components with invalid attributes', () => {
+      // React 15.2 warns when setting a non valid prop to an DOM element
+      sandbox.stub(console, 'error');
```

Comment:
The test will pass, it will simply warn in the console.
![image](https://cloud.githubusercontent.com/assets/5271169/17693057/5fcac1da-639d-11e6-878c-680c5ae826c4.png)